### DISCUSSION TITLE
Anyone know how to add GPSR compliance data to Etsy listings with the API?

### DISCUSSION THREAD

**Discussion by magnalister** [etsy/open-api#1307] [Q&A]
I want to add the GPSR (General Product Safety Regulation) information to my Etsy listings, but I'm not quite sure how to do this via Etsy's API. 

Has anyone done this before or have any tips?

**Comment by Wilderbill** [comment]
Download a listing.
Modify the Description with the addition of your GPRS.
Upload the listing.

**Comment by alexh-swdev** [comment]
There are new inputs for GPRS manufacturer and safety infos in the back-end.
When are these properties available in the API to bulk-edit them instead of doing this for every product manually?

/Edit: Or is there at least a way in the back-end to set this info globally for all items?

**Comment by chasemorell** [comment]
Is there an expected timeline for when the API will support the ability to edit GPSR compliance data? This functionality would be extremely valuable.
